handlers/rsa.go

```diff
@@ -0,0 +1,211 @@
+package handlers
+
+import (
+	"crypto"
+	"crypto/dsa"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/pem"
+	"net/http"
+	"path/filepath"
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/vocdoni/blind-csp/csp"
+	"go.vocdoni.io/dvote/db"
+	"go.vocdoni.io/dvote/db/metadb"
+	"go.vocdoni.io/dvote/log"
+)
+
+var rsaPubKeys = []string{
+	`-----BEGIN PUBLIC KEY-----
+MIIBIDANBgkqhkiG9w0BAQEFAAOCAQ0AMIIBCAKCAQEApc2hU8zulyJzdQE5IPAv
+B2BgveoZmYUmPEjSb4DViBoATK1hlaY8Psp5vj0H0L4tM8AlXRhPQlECibhgccig
+xQFcG7CLXiSAn7c4XoR+J2SCgx76Fwl9L3WhQigxyKsmpGIqubseydmwfJi4TBnq
+qnX4prsW1PT8GpG35t8Qi8PtkXVGmL7G5pkPXtF0hRzKSfhzsDBbJsl6Jk/Rn5Id
+pKHXL22FdbE9fGzIlW2a6Zdd0b0Q3FZBMnWLSwo0OwBtC/qNnDTCzboig9djiFmA
+yuj8jVhsy050nI72TAONjGKi+xn4lYfdOV2k6TyvpRHfylHouK2v0/bktSlkFI0y
+nwIBAw==
+-----END PUBLIC KEY-----`,
+}
+
+// RsaHandler is a handler that allows only 1 registration for IP
+type RsaHandler struct {
+	kv       db.Database
+	keysLock sync.RWMutex
+}
+
+func (rh *RsaHandler) addKey(index, value []byte) {
```

Comment:
This method should return an error, as we do on this production handler (not dummy) https://github.com/vocdoni/blind-csp/blob/master/handlers/idcat.go#L88

Reply:
Done